### DISCUSSION TITLE
Build cpp on windows is broken

### DISCUSSION THREAD

**Discussion by gengjiawen** [community/community#25858] [Actions]
<p><a href="https://github.com/gengjiawen/uvwasi/runs/255955641" rel="nofollow noopener">https://github.com/gengjiawen/uvwasi/runs/255955641</a></p>
<p>I don’t know if the windows has cmake, but it clearly not run the cmake command.</p>

**Comment by chrispat** [comment, marked as answer]
<p>this is becaue npx exited the cmd.exe process try putting the npx call in its own run step or creating a conditional step for windows that uses powershell as the default shell.</p>

**Comment by gengjiawen** [comment]
<p>How should I fix this ? Any link I can check ?</p>

**Comment by gengjiawen** [comment]
<p><a href="https://github.com/gengjiawen/os-init/pull/3/files" rel="nofollow noopener">https://github.com/gengjiawen/os-init/pull/3/files</a></p>
<p>separate looks like works.</p>
